### PR TITLE
[FLINK-34687][cdc][docs] Build the home page of Flink CDC documentation

### PR DESCRIPTION
**Pull request by leonardBang**
This PR build a new home page  of Flink CDC documentation to help user know this project easily .

### CODE REVIEW

**Comment by xleoken** [comment]
👍

File: docs/content/_index.md [hunk 1]
```diff
@@ -22,17 +22,179 @@ specific language governing permissions and limitations
...
+</div>
+
+<div style="display: flex;">
+    <br><br><br><br>
+    <div style="flex: 5%;"></div>
+    <div style="text-align: center; flex: 25%;">
+          <p style="text-align: center; color: #BF74F1; font-size: large; padding: 0">Change Data Capture</p>
+          <hr style="background-color:#BF74F1; width: 60%">
+          <p style="text-align: left;">
+            FLink CDC supports distributed scanning of historical data of database and then automatically switches to change data capturing. The switch uses the incremental snapshot algorithm which ensure the switch action does not lock the database.
```
Comment (PatrickRen) [inline docs/content/_index.md:94]:
```suggestion
            Flink CDC supports distributed scanning of historical data of database and then automatically switches to change data capturing. The switch uses the incremental snapshot algorithm which ensure the switch action does not lock the database.  
```

File: docs/content/_index.md [hunk 2]
```diff
@@ -22,17 +22,179 @@ specific language governing permissions and limitations
...
+          <p style="text-align: left;">
+            FLink CDC supports distributed scanning of historical data of database and then automatically switches to change data capturing. The switch uses the incremental snapshot algorithm which ensure the switch action does not lock the database.  
+          </p>
+    </div>
+    <div style="flex: 8%;"></div>
+    <div style="text-align: center; flex: 24%;">
+          <p style="text-align: center; color: #BF74F1; font-size: large; padding: 0">Schema Evolution</p>
+          <hr style="background-color:#BF74F1; width: 60%">
+          <p style="text-align: left;">
+            Flink CDC will automatically create downstream table using the inferred table structure based on upstream table. Flink CDC will automatically apply upstream DDL to downstream systems during change data capturing.</p>
```
Comment (PatrickRen) [inline docs/content/_index.md:102]:
```suggestion
            Flink CDC has the ability of automatically creating downstream table using the inferred table structure based on upstream table, and applying upstream DDL to downstream systems during change data capturing.</p>
```

File: docs/content/_index.md [hunk 3]
```diff
@@ -22,17 +22,179 @@ specific language governing permissions and limitations
...
+    </div>
+    <div style="flex: 8%;"></div>
+    <div style="text-align: center; flex: 25%;">
+          <p style="text-align: center; color: #BF74F1; font-size: large; padding: 0">Streaming Pipeline</p>
+          <hr style="background-color:#BF74F1; width: 60%">
+          <p style="text-align: left;">
+            Flink CDC jobs run in streaming mode by default, providing sub-second end-to-end latency in real-time binlog synchronization scenarios, this feature effectively ensures data freshness for downstream businesses.</p>
```
Comment (PatrickRen) [inline docs/content/_index.md:109]:
```suggestion
            Flink CDC jobs run in streaming mode by default, providing sub-second end-to-end latency in real-time binlog synchronization scenarios, effectively ensuring data freshness for downstream businesses.</p>
```

File: docs/content/_index.md [hunk 4]
```diff
@@ -22,17 +22,179 @@ specific language governing permissions and limitations
...
+          <p style="text-align: center; color: #BF74F1; font-size: large; padding: 0">Data Transformation</p>
+          <hr style="background-color:#BF74F1; width: 60%">
+          <p style="text-align: left;">
+            Flink CDC will soon support data transform operations of ETL, including column projection, computed column, filter expression and classical scalar functions.</p>
+    </div>
+    <div style="flex: 8%;"></div>
+    <div style="text-align: center; flex: 24%;">
+          <p style="text-align: center; color: #BF74F1; font-size: large; padding: 0">Full Database Sync</p>
+          <hr style="background-color:#BF74F1; width: 60%">
+          <p style="text-align: left;">Flink CDC supports synchronizing all tables of source database instance to downstream in one job, user can config the captured database list and table list.</p>
```
Comment (PatrickRen) [inline docs/content/_index.md:131]:
```suggestion
          <p style="text-align: left;">Flink CDC supports synchronizing all tables of source database instance to downstream in one job by configuring the captured database list and table list.</p>
```

File: docs/content/_index.md [hunk 5]
```diff
@@ -22,17 +22,179 @@ specific language governing permissions and limitations
...
+</div>
+
+<div style="display: flex;">
+    <br><br><br><br>
+    <div style="flex: 1;">
+    </div>
+    <div style="text-align: left; flex: 3.5; width: 100%">
+          <p style="text-align: left; color: #BF74F1; font-size: x-large; padding: 0">Try Flink CDC</p>
+          <p style="text-align: left;">
+        Flink CDC provides a series of quick start demos without any dependencies or java code, only a Linux or MacOS computer with Docker installed is enough.
```
Comment (PatrickRen) [inline docs/content/_index.md:167]:
```suggestion
        Flink CDC provides a series of quick start demos without any dependencies or java code. A Linux or MacOS computer with Docker installed is enough. 
```

File: docs/content/_index.md [hunk 6]
```diff
@@ -22,17 +22,179 @@ specific language governing permissions and limitations
...
+        Flink CDC provides a series of quick start demos without any dependencies or java code, only a Linux or MacOS computer with Docker installed is enough. 
+        Please check out our <a href="docs/get-started/introduction">Quick Start</a> for more information.
+         </p>
+    </div>
+    <div style="flex: 1;"></div>
+    <div style="text-align: left; flex: 3.5; width: 100%">
+          <p style="text-align: left; color: #BF74F1; font-size: x-large; padding: 0">Get Help with Flink CDC</p>
+          <p style="text-align: left;">
+            If you get stuck, check out our <a href="https://flink.apache.org/community.html">community support resources</a>. 
+                In particular, Apache Flink’s user mailing list is consistently ranked as one of the most active of
```
Comment (PatrickRen) [inline docs/content/_index.md:176]:
```suggestion
                In particular, Apache Flink’s user mailing list (user@flink.apache.org) is consistently ranked as one of the most active of
```

File: docs/content/_index.md [hunk 7]
```diff
@@ -22,17 +22,179 @@ specific language governing permissions and limitations
...
 
-#### 
+<div style="background-image: url('fig/index-background-header.png'); size:inherit; background-size: auto 100%; overflow: hidden">
+    <div style="text-align: center">
+        <br><br><br><br><br><br>
+        <h1 style="color: #FFFFFF">
+            Flink CDC
+        </h1>
+        <h3 style="color: #FFFFFF">
+            The streaming data integration tool
```
Comment (PatrickRen) [inline docs/content/_index.md:32]:
```suggestion
            A streaming data integration tool
```
"The" feels like we are the only one lol

**Review by PatrickRen — COMMENTED** [review]
@leonardBang Thanks for the PR! I left some comments

**Review by PatrickRen — APPROVED** [review]
@leonardBang Thanks for the update! LGTM

**Pull request merged**